tests/src/data/use-cases/db-create-access-token.spec.ts

```diff
@@ -77,7 +76,7 @@ describe('DbCreateAccessToken UseCase', () => {
 
       await sut.create(fakeEmail)
 
-      expect(spyTokenService).toHaveBeenCalledWith({ sub: mockFakeGuardianLoaded().id })
+      expect(spyTokenService).toHaveBeenCalledWith({ sub: 'any_id' })
```

Comment:
Maybe use a global variable to this mocks are useful, just to omit magics values like 'any_id'. What you think?

Reply:
it makes sense to do that